usr/log.c

```diff
@@ -336,6 +336,20 @@ void log_info(const char *fmt, ...)
 	va_end(ap);
 }
 
+void log_session_reopen(int level, iscsi_session_t *session, const char *fmt, ...)
```

Comment:
I dislike that we need a new function just for this one case. Maybe it should be called something more generic, in case we want to use it elswhere, like "log_debug_limited()" or "log_debug_freq_limit()" I'd rather have a long name than an obscure one. But it needs "debug" in the title, since normally debug logging is set to 0 (none) for many installations.

It looks like 6 messages (if I count correctly) have changed from log_debug() to log_session_reopen(). How did you pick those? No, I see one was a "log_error()".  Isn't that the one that really started all this, i.e. log_error("connect to %s:%s failed (%s)");.